dist/srpm/rhc.spec.in

```diff
@@ -122,12 +116,25 @@ ln -sf ../yggdrasil/config.toml %{buildroot}%{_sysconfdir}/rhc/config.toml
 %endif
 %endif
 
+%pre
+if [ $1 -eq 2 ]; then
+    if [ -f /etc/rhc/config.toml ]; then
+        cp /etc/rhc/config.toml /etc/yggdrasil/config.toml.migrated
+    fi
+fi
+
 %post
 %systemd_post rhc-canonical-facts.timer
 if [ $1 -eq 1 ]; then
      systemctl daemon-reload
      systemctl start rhc-canonical-facts.timer
 fi
+if [ $1 -eq 2 ]; then
+    if [ -f /etc/yggdrasil/config.toml.migrated ]; then
+        sed -e 's#broker#server#g' /etc/yggdrasil/config.toml.migrated > /etc/yggdrasil/config.toml
```

Comment:
Hmm I think this sed expression is a bit too broad, and there's the risk of it changing also other parts than the wanted bit (imagine if the server hostname contains "broker"...), e.g. comments, etc. Maybe at least `^` to check for the beginning of the line, `=` for the key/value separator, and white spaces allowed according to what TOML supports.